### ISSUE TITLE
Lets use common phrasing

### ISSUE THREAD

**Issue by IanKWatts** [bcgov/test1#12]
## TL;DR 🏎️

Teams are encouraged to favour modern inclusive phrasing both in their communication as well as in any source checked into their repositories. You'll find a table at the end of this text with preferred phrasing to socialize with your team.

## Words Matter

We're aligning our development community to favour inclusive phrasing for common technical expressions. There is a table below that outlines the phrases that are being retired along with the preferred alternatives.

During your team scrum, technical meetings, documentation, the code you write, etc. use the inclusive phrasing from the table below. That's it - it really is that easy.

For the curious mind, the Public Service Agency (PSA) has published a guide describing how [Words Matter](https://www2.gov.bc.ca/assets/gov/careers/all-employees/working-with-others/words-matter.pdf) in our daily communication. Its an insightful read and a good reminder to be curious and open minded.

## What about the master branch?

The word "master" is not inherently bad or non-inclusive. For example people get a masters degree; become a master of their craft; or master a skill. It's generally when the word "master" is used along side the word "slave" that it becomes non-inclusive.

Some teams choose to use the word `main` for the default branch of a repo as opposed to the more commonly used `master` branch. While it's not required or recommended, your team is empowered to do what works for them. If you do rename the `master` branch consider using `main` so that we have consistency among the repos within our organization. 

## Preferred Phrasing

| Non-Inclusive  |    | Inclusive |
| :------------- |:--:| :-------- |
| Whitelist      | => | Allowlist |
| Blacklist      | => | Denylist |
| Master / Slave | => | Leader / Follower; Primary / Standby; etc |
| Grandfathered  | => | Legacy status |
| Sanity check   | => | Quick check; Confidence check; etc |
| Dummy value    | => | Placeholder value; Sample value; etc |

### Pro Tip 🤓

This list is not comprehensive. If you're aware of other outdated nomenclature please create an issue (PR preferred) with your suggestion.
